<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>autobots_toutiaoClass</class>
 <widget class="QMainWindow" name="autobots_toutiaoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>295</width>
    <height>138</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>人物投票</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="pushButton_start">
        <property name="text">
         <string>开始</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_pause">
        <property name="text">
         <string>停止</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_1">
        <property name="text">
         <string>当前状态</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_msg">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>295</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_from_txt">
   <property name="text">
    <string>导入账号</string>
   </property>
   <property name="toolTip">
    <string>txt文本</string>
   </property>
  </action>
  <action name="action_import_commentid">
   <property name="text">
    <string>导入评论列表</string>
   </property>
  </action>
  <action name="action_clear_comments">
   <property name="text">
    <string>清空评论列表</string>
   </property>
  </action>
  <action name="action_clear_accounts">
   <property name="text">
    <string>清空账号列表</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="autobots_toutiao.qrc"/>
 </resources>
 <connections/>
</ui>
